### ISSUE TITLE
Wrong decimal separator on serialization

### ISSUE THREAD

**Issue by OrthFelix** [OfficeDev/Open-XML-SDK#911]
**Description**

In cultures that use the decimal comma as decimal separator, floating point numbers get serialized wrongly.

**Information**

- .NET Target: .NET Standard 2.0 or .NET 5
- DocumentFormat.OpenXml Version: 2.12.3
- Language is German in this case

**Repro**

Example is with .NET 5 in language version C# 9.0 for shorter code
```csharp
using DocumentFormat.OpenXml;
using DocumentFormat.OpenXml.Packaging;
using DocumentFormat.OpenXml.Spreadsheet;

var spreadsheetDocument = SpreadsheetDocument.Create("demo.xlsx", SpreadsheetDocumentType.Workbook);

var workbookpart = spreadsheetDocument.AddWorkbookPart();
var worksheetPart = workbookpart.AddNewPart<WorksheetPart>();

worksheetPart.Worksheet = new Worksheet(
    new SheetData(
        new Row(
            new Cell
            {
                CellValue = new CellValue(0.1),
                DataType = new EnumValue<CellValues>(CellValues.Number)
            })));

workbookpart.Workbook = new Workbook(
    new Sheets(
        new Sheet()
        {
            Id = spreadsheetDocument.WorkbookPart.GetIdOfPart(worksheetPart),
            SheetId = 1,
            Name = "Demo"
        }));

spreadsheetDocument.Close();
``` 

**Observed**

Value gets serialized with a decimal comma and results in a broken xlsx

**Expected**

Value gets serialized with a decimal point

**Comment by tomjebo** [comment]
@OrthFelix Thanks for reporting this. It should be fixed in the 2.13.0-beta0001 version on Nuget. We added this fix in #904. Can you test with 2.13.0-beta0001?

**Comment by OrthFelix** [comment]
Sure, it seems to be working!
